docker/pwsh-exchange/build.conf

```diff
@@ -1 +1 @@
-version=1.0.0
+version=7.1.1
```

Comment:
No need for this bump. We only bump the version to the base images.

Reply:
I thought that we should match the docker version to the pwsh version.